src/Renderer/LayeredMaterial.js

```diff
@@ -456,23 +477,31 @@ LayeredMaterial.prototype.getColorLayerLevelById = function getColorLayerLevelBy
     return texture ? texture.coords.zoom : EMPTY_TEXTURE_ZOOM;
 };
 
+LayeredMaterial.prototype.isColorLayerLoaded = function isColorLayerLoaded(layer) {
+    const textures = this.getLayerTextures(layer);
+    if (textures.length) {
+        return textures[0].coords.zoom > EMPTY_TEXTURE_ZOOM;
+    }
+    return false;
+};
+
 LayeredMaterial.prototype.getElevationLayerLevel = function getElevationLayerLevel() {
     return this.textures[l_ELEVATION][0].coords.zoom;
 };
 
-LayeredMaterial.prototype.getLayerTextures = function getLayerTextures(layerType, layerId) {
-    if (layerType === l_ELEVATION) {
+LayeredMaterial.prototype.getLayerTextures = function getLayerTextures(layer) {
+    if (layer.type === 'elevation') {
```

Comment:
Side note: shouldn't it be more practical to change the constants in `LayeredMaterialConstants` to `elevation`, `color` and all ?

Reply:
IMHO, `LayeredMaterialConstants` should be removed soon because that should be an implementation details.

But we'll also need to have valid layer types as constants somewhere (`elevation`, `color`, etc).